pipelinewise/cli/alert_handlers/base_alert_handler.py

```diff
@@ -0,0 +1,28 @@
+"""
+PipelineWise CLI - Base class of alert handlers
+"""
+
+
+# pylint: disable=too-few-public-methods
+class BaseAlertHandler:
```

Comment:
I recommend configuring this class to be an abstract class with the help `from abc import ABC` and then decorating `send` with `@abstractmethod`.
That way nobody can create direct instances of `BaseAlertHandler` and any class that  inherits from `BaseAlertHandler` would be forced to implement `send` and avoid getting `NotImplementedError` at runtime.

Reply:
Check the doc https://docs.python.org/3/library/abc.html

Reply:
thanks, addressed by https://github.com/transferwise/pipelinewise/pull/467/commits/d4cd6e022a4afa7414d5f4055c571520458905db